portal_customer_wallet/tests/common.py

```diff
@@ -0,0 +1,134 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import datetime
+
+from odoo.tests.common import TransactionCase
+
+
+class TestCommon(TransactionCase):
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super().setUpClass(*args, **kwargs)
+
+        cls.partner_id = cls.env["res.partner"].create({"name": "Test Partner"})
+        cls.company_id = cls.env.ref("base.main_company")
+        cls.pricelist_id = cls.partner_id.property_product_pricelist
+        cls.customer_wallet_account = cls.env["account.account"].create(
+            {
+                "name": "Test Wallet Account",
+                "code": "10101010101",
+                "account_type": "liability_current",
+            }
+        )
+        cls.company_id.customer_wallet_account_id = cls.customer_wallet_account
+        cls.customer_wallet_journal = cls.env["account.journal"].create(
+            {
+                "name": "Test Wallet Journal",
+                "code": "TSTWLLT",
+                "type": "bank",
+                "is_customer_wallet_journal": True,
+                "inbound_payment_method_line_ids": [
+                    (
+                        0,
+                        False,
+                        {
+                            "payment_method_id": cls.env.ref(
+                                "account.account_payment_method_manual_in"
+                            ).id,
+                            "name": "Manual",
+                            "payment_account_id": cls.customer_wallet_account.id,
+                        },
+                    )
+                ],
+                "outbound_payment_method_line_ids": [
+                    (
+                        0,
+                        False,
+                        {
+                            "payment_method_id": cls.env.ref(
+                                "account.account_payment_method_manual_out"
+                            ).id,
+                            "name": "Manual",
+                            "payment_account_id": cls.customer_wallet_account.id,
+                        },
+                    )
+                ],
```

Comment:
it seems strange to me to have to manually create the `inbound_payment_method_line_ids` and `outbound_payment_method_line_ids` which are computed fields. isn’t there a better way to do this? how is it done in normal usage?

Reply:
I've adjusted this to match the normal usage, which is to adjust one field (payment_account_id) on the record that is created by default.